test/e2e/specs/support/support__home.ts

```diff
@@ -54,11 +54,13 @@ describe( DataHelper.createSuiteTitle( 'Support: My Home' ), function () {
 
 		// Invalid keyword search often takes more than 30s to resolve.
 		// See: https://github.com/Automattic/wp-calypso/issues/55478
+		// eslint-disable-next-line jest/no-disabled-tests
```

Comment:
I wonder if we should just disable this rule altogether.

Reply:
What I will do, since this PR has gone on for long enough:
1. merge this PR with the disable statements included;
2. spin up a new issue to remove or disable this ESLint rule entirely;
3. remove all instances of this rule being disabled in the repo.

I'll likely do this as my next 20% project.

Reply:
Nice, thanks 👍